app/policies/event_policy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class EventPolicy < ApplicationPolicy
```

Comment:
This is the policy that applies to our `Event` model - to perform any controller action, the user will have to be an admin.